bundles/org.openhab.binding.miio/src/main/resources/database/zhimi.heater.zb1-miot.json

```diff
@@ -85,6 +113,12 @@
 				"channelType": "time",
```

Comment:
Why did you remove `"channelType": "time",` from some of the devices but not others?

Reply:
I nowadays remove the channeltype (unless very specific reasons) because than it is bit more transparant and less error prone.

In this case... it was just because I missed removing this one. :-(